docs/source/conventions/example.rst

```diff
@@ -145,75 +109,35 @@ labs).
 
 The VRS computed identifier procedure requires that all nested
 :term:`identifiable objects <identifiable object>` are expressed using
-computed identifiers.  Using GA4GH sequence identifiers collapses
+computed identifiers. Using GA4GH sequence identifiers collapses
 differences between alleles due to trivial differences in reference
-naming.  The same variation reported on NC_000019.10, CM000681.2,
+naming. The same variation reported on NC_000019.10, CM000681.2,
 GRCh38:19, GRCh38.p13:19 would appear to be distinct variation; using
-a digest identifer will ensure that variation is reported on a single
-sequence identifier.  Furthermore, using digest-based sequence
+a digest identifier will ensure that variation is reported on a single
+sequence identifier. Furthermore, using digest-based sequence
 identifiers enables the use of custom reference sequences.
 
-.. important:: VRS permits the use of conventional sequence accessions
-	       from RefSeq, Ensemble, or other sources.  However, when
-	       generating copmuted identifiers, implementations MUST
-	       use GA4GH-sequence accessions.
-
-In this example, the sequence identifier ``refseq:NC_000019.10`` MUST
-be transformed into digest-based identifer
-``ga4gh:GS.IIB53T8CNeJJdUqzn9V_JnRtQadwWCbl`` as described in
-:ref:`computed-identifiers`.  In practice, implmentations should
-precompute sequence digests or should use an existing service that
-does so. (See :ref:`required-data` for a description of data that are
-needed to implement VRS.) Subsitituing the GA4GH sequence identifier
-into the Allele's ``location.sequence_id`` attribute gives:
```

Comment:
Another one... I just removed the entire section. Did we want to say anything in particular about RefGet accessions being required now?

Reply:
I think here we could just update the important box to talk about how you can annotate `SequenceReference` with common accessions as well as GA4GH Digest info.